.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: "required"
+dist: "trusty"
+language: "generic"
+services:
+  - "docker"
+script:
+  - "bash build.sh"
+deploy:
+  provider: "script"
+  script: "bash docker_push.sh"
```

Comment:
Seems like there is soom for a nasty timing issue, since both build.sh and docker_push.sh use `DREDD_VERSION=$(npm view dredd version)` to get the Dredd version, what if the latest version were to change between the execution of the two scripts? I think it would be better to pass it to both scripts and compute once.

Reply:
https://github.com/apiaryio/dredd-docker/pull/2